apps/OboeTester/app/src/main/cpp/util/WaveFileWriter.h

```diff
@@ -15,7 +15,7 @@
  */
 
 // Based on the WaveFileWriter in Java from the open source JSyn library by Phil Burk
-// https://github.com/philburk/jsyn/blob/master/src/com/jsyn/util/WaveFileWriter.java
+// https://github.com/philburk/jsyn/blob/master/src/main/java/com/jsyn/util/WaveFileWriter.java
```

Comment:
JSyn is still using a "master" branch.

Reply:
Yup, hence this is https://github.com/philburk/jsyn/blob/master/src/main/java/com/jsyn/util/WaveFileWriter.java and not https://github.com/philburk/jsyn/blob/master/src/master/java/com/jsyn/util/WaveFileWriter.java

Reply:
Ah! So there is a "main" folder in the "master" branch in JSyn. Got it.